get-started/welcome.qmd

```diff
@@ -0,0 +1,91 @@
+---
+title: "Welcome"
+subtitle: Effortless Python web applications
+
+listing:
+  - id: gallery-highlight
+    template: gallery-highlight.ejs
+    contents: gallery-highlight.yml
+
+editor:
+  markdown:
+    wrap: sentence
+---
+
+Create efficient, reactive, and robust web applications and dashboards.
+
+![Code of example shiny application](assets/code-welcome.png){class="img-shadow"}
+
+```python
+# | standalone: true
+
+from shiny.express import ui, render, input
+import seaborn as sns
+
+df = sns.load_dataset("penguins")
+
+ui.input_slider("bins", "Number of Bins", min=5, max=20, value=15, step=1)
+
+
+@render.plot
+def plot():
+    ax = sns.histplot(data=df, x="body_mass_g", bins=input.bins())
+```
+
+
+## Why Shiny
+
+- **No JavaScript required** – Build full-featured interactive apps in pure Python.
+  [Install](install.qmd) and [create your first app](create-run.qmd) now!
+- **Fast and robust iteration** – The reactive model allows efficient and dynamic updates.
+  [Learn more about reactivity](/docs/reactive-foundations.qmd).
+- **Batteries Included** – Comes with everything you need for your applications and dashboards.
+  Check out the [input](/components/#inputs) and [output](/components/#outputs) you can use in the [components gallery](/components/) and the different ways you can arrange elements with [layouts](/layouts/).
+- **Scalability and extensibility** – Start simple, grow as needed. [Learn more about custom components](/docs/custom-component-one-off.qmd).
+- **Starter templates** - Get up and running quickly with a [starter template](/templates/).
+
+Shiny for Python empowers you to bring your data to life with interactive applications that are easy to [build](create-run.qmd), [customize](/docs/ui-customize.qmd), and [share](deploy.qmd).
+
+## Everything you need
+
+Need an interactive [component](/components/) on the page?
+We got you.
+
+<!--
+Sample of component gallery mini docs that link to component
+-->
+
+- slider
+- selectize
+- date
+- map
+- plotly
+- valuebox
```

Comment:
this section here would be a few examples of inputs and outputs that look like small cards like the components gallery.

Reply:
I cut this, I think there's enough going on in this page. We can come back to add it later if we want.